src/index.js

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable import/no-mutable-exports */
+import './style.css';
+
+const submit = document.querySelector('.add-btn');
+const tasksDiv = document.querySelector('.tasks');
+export const taskInput = document.querySelector('.add-task input');
+
+let editId;
+let isEditTask = false;
+let tasks = JSON.parse(localStorage.getItem('task-list'));
+
+export const addElemToPage = () => {
+  let div = '';
+  if (tasks) {
+    tasks.forEach((task) => {
+      let isCompleted;
+      if (task.completed === true) {
+        isCompleted = 'checked';
+      } else {
+        isCompleted = '';
+      }
```

Comment:
What do you think of getting the id of the checkbox and comparing it to the index of the task to make your code look shorter?